gdx/src/com/badlogic/gdx/scenes/scene2d/Actor.java

```diff
@@ -612,12 +616,17 @@ public float getRotation () {
 	}
 
 	public void setRotation (float degrees) {
-		this.rotation = degrees;
+		if (this.rotation != degrees) {

```

Comment:
`setWidth()` and `setHeight()` save the old value and then compare it with the new to see whether or not to call the handler. Can we do it the same way just for consistency?